recipes/psy-view/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "psy-view" %}
+{% set version = "0.0.1b0" %}
```

Comment:
We don't generally accept beta versions, but have not had consistent policies either. @conda-forge/core what do we do here again?

Reply:
Thanks for the info @beckermr! The first stable version will come pretty soon as the basis die the software (psyplot, etc.) is already around for quite a while. We just made this a beta version to signalize that the layout of the GUI and implemented features are subject to discussion with the community.

Reply:
Understood. We dealt with this case recently and have a policy now. I am simply forgetting it.

Reply:
@beckermr, is this what you are looking for?

https://github.com/conda-forge/cfep/blob/master/cfep-05.md#exceptions

Reply:
seems to me like it is generally allowed to publish pre-release packages on the conda-forge channel, as long there has not yet been a non-beta/alpha release, yet (which applies for psy-view). As soon as there is a non-beta/alpha release for the package, pre-releases need to get special labels